exporters/trace/stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandler.java

```diff
@@ -239,6 +249,90 @@ private static Attributes toAttributesProto(
     return attributesBuilder;
   }
 
+  @SuppressWarnings("unchecked")
+  private static void putResourceLabels(
+      Attributes.Builder attributesBuilder, MonitoredResource resource) {
+    if (resource == null) {
+      return;
+    }
+    ResourceType resourceType = resource.getResourceType();
+    switch (resourceType) {
+      case AWS_EC2_INSTANCE:
+        AwsEc2InstanceMonitoredResource awsEc2InstanceMonitoredResource =
+            (AwsEc2InstanceMonitoredResource) resource;
+        attributesBuilder.putAttributeMap(
+            createResourceLabelKey(resourceType, "aws_account"),
+            toStringAttributeValueProto(awsEc2InstanceMonitoredResource.getAccount()));
```

Comment:
>         attributesBuilder.putAttributeMap(
>             createResourceLabelKey(resourceType, "aws_account"),
>             toStringAttributeValueProto(awsEc2InstanceMonitoredResource.getAccount()));

Suggestion: The code that adds an attribute to the builder could probably be factored out into a helper method.